roles/rdbms-setup/tasks/rdbms-install.yml

```diff
@@ -128,6 +128,9 @@
   become: yes
   become_user: "{{ oracle_user }}"
   command: "{{ oracle_home }}/OPatch/{{ item.method }} -silent -oh {{ oracle_home }} {{ swlib_unzip_path }}/{{ item.patchnum }}{{ item.patch_subdir }}"
+  environment:
+    PATH: "{{ oracle_home }}/perl/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin"
```

Comment:
Same as above comment.
Is it possible to extract existing PATH to a var and append after `{{ oracle_home }}/perl/bin`?

Reply:
To be honest I'd rather not.  This syntax is consistent with most other places in the toolkit (https://github.com/google/bms-toolkit/blob/e1a7eba1e1241231d9831252a2251b8928dc0f0e/roles/patch/tasks/rac-dbpatch.yml#L21 https://github.com/google/bms-toolkit/blob/e1a7eba1e1241231d9831252a2251b8928dc0f0e/roles/db-copy/tasks/active-copy.yml#L43 https://github.com/google/bms-toolkit/blob/62bf71da213a5799d95a46e925e34a48af371a9e/roles/brute-ora-cleanup/tasks/main.yml#L32 for example) and seems to work pretty well.  I'd prefer not to add yet another convention.

Reply:
Ack, lgtm.

Can you please post the o/p in a gist after running the install after the latest commit [fc12ade](https://github.com/google/bms-toolkit/pull/138/commits/fc12aded78b78c1a08b6492e65b1fb87ab432e9f)?

Thanks